public/plugin.ts

```diff
@@ -61,6 +71,117 @@ export class notificationsDashboardsPlugin
       });
     }
 
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
+
+      // register applications with category and use case information
+      core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
```

Comment:
Notification features should be only put into settings and setup nag group.

Reply:
@amsiglan  ^^

Reply:
addressed